examples/quickstarts/nodejs-mongodb-persistent.json

```diff
@@ -102,7 +102,7 @@
             "from": {
               "kind": "ImageStreamTag",
               "namespace": "${NAMESPACE}",
-              "name": "nodejs:4"
+              "name": "nodejs:6"
```

Comment:
you can't update these files directly:
https://github.com/openshift/origin/blob/master/examples/quickstarts/README.md

maybe i need to put that note at the top instead of the bottom of the file.